src/app/modules/rup/components/ejecucion/hudsBusqueda.html

```diff
@@ -435,6 +435,13 @@
                                                     <div class="row p-0 m-0">
                                                         <div class="col-10 p-0 m-0">
                                                             <div class="sugerido">
+                                                                <ng-container *ngIf="prestacion.tipo === 'ficha-epidemiologica'">
+                                                                    <small>
+                                                                        Confirmación Final:
+                                                                        {{ prestacion.confirmacion }}
+                                                                    </small>
```

Comment:
Ahí lo charlamos en UI/UX, lo mejor por lo pronto es esto:
```suggestion
<plex-badge size="sm" type="info">clasificación: {{ prestacion.confirmacion }}</plex-badge>

```

Reply:
Hecho!
![image](https://user-images.githubusercontent.com/25707055/127198593-d8a376c7-95b9-49ed-9f36-7ea040ad9a95.png)

Reply:
Opinión personal, debería ser un badge al lado de VALIDADA y que diga el estado **solo**. Con un tooltip que diga _clasificación_.

Reply:
Usamos algún código de color? para diferenciar los tipos de clasificación (confirmado, descartado, etc) o color info para todo?